zproject/prod_settings_template.py

```diff
@@ -20,11 +20,11 @@
 # See http://zulip.readthedocs.io/en/latest/prod-authentication-methods.html
 # for documentation on our authentication backends.
 AUTHENTICATION_BACKENDS = (
-#    'zproject.backends.EmailAuthBackend', # Email and password; see SMTP setup below
-#    'zproject.backends.GoogleMobileOauth2Backend', # Google Apps, setup below
-#    'zproject.backends.GitHubAuthBackend', # GitHub auth, setup below
-#    'zproject.backends.ZulipLDAPAuthBackend', # LDAP, setup below
-#    'zproject.backends.ZulipRemoteUserBackend', # Local SSO, setup docs on readthedocs
+    #    'zproject.backends.EmailAuthBackend', # Email and password; see SMTP setup below
+    #    'zproject.backends.GoogleMobileOauth2Backend', # Google Apps, setup below
+    #    'zproject.backends.GitHubAuthBackend', # GitHub auth, setup below
+    #    'zproject.backends.ZulipLDAPAuthBackend', # LDAP, setup below
+    #    'zproject.backends.ZulipRemoteUserBackend', # Local SSO, setup docs on readthedocs
```

Comment:
I think you should instead have the lines look like 

```
+    # 'zproject.backends.ZulipRemoteUserBackend', # Local SSO, setup docs on readthedocs
```

(with less space after the `#`).